homeassistant/components/volvooncall/__init__.py

```diff
@@ -169,6 +170,15 @@ def instrument(self, vin, component, attr, slug_attr):
             None,
         )
 
+        if ret is None:
+            _LOGGER.warning(
```

Comment:
Raise an exception instead, eg by not using a default in `next`. This is a developer error.